modules/activemq-artemis-launch/added/drain.sh

```diff
@@ -71,4 +110,17 @@ else
 fi
 
 waitForJolokia
-curl -s -o /dev/null -G -k http://${AMQ_USER}:${AMQ_PASSWORD}@${BROKER_IP}:8161/console/jolokia/exec/org.apache.activemq.artemis:broker=%22${AMQ_NAME}%22/scaleDown/scaledownconnector
+
+RET_CODE=`curl -G -k http://${AMQ_USER}:${AMQ_PASSWORD}@${BROKER_IP}:8161/console/jolokia/exec/org.apache.activemq.artemis:broker=%22${AMQ_NAME}%22/scaleDown/scaledownconnector`
+
+HTTP_CODE=`echo $RET_CODE | python2 -c "import sys, json; print json.load(sys.stdin)['status']"`
+
+echo "curl return code ${HTTP_CODE}"
+
+if [ "${HTTP_CODE}" != "200" ]; then
+  echo "scaleDown is not successful, response: $RET_CODE sleep for debug"
+  exit 1
+fi
+
+echo "scaledown is successful sleep for debug"
```

Comment:
I don't see any sleep command, why `sleep for debug`?

Reply:
Probably so the container stays up ...?

Reply:
yes that's something I forgot to remove. :)